hunting/windows/queries/detect_dll_hijack_via_masquerading_as_microsoft_native_libraries_elastic_defend.toml

```
@@ -0,0 +1,31 @@
+[hunt]
+author = "Elastic"
+integration = "logs-endpoint.events.library-*"
+uuid = "87c97865-fdaa-48b2-bfa6-67bed7cf56ef"
+name = "Detect DLL Hijack via Masquerading as Microsoft Native Libraries - Elastic Defend"
+language = "ES|QL"
+license = "Elastic License v2"
+notes = [
+    "This hunt require the creation of an enrichment policy to use with the ES|QL (ENRICH command).",
+    "Using dll.hash.sha256 for Elastic Defend or file.hash.sha256 for Sysmon you can pivot to further investigate the DLL origin and purpose.",
```

```suggestion
    "The `dll.hash.sha256` field can be used to pivot and further investigate the DLL origin and purpose.",
```

Removing sysmon stuff as this one is elastic defend specific.